_posts/2017-09-01-sept-01.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "This Week In Erlang Sept 1"
+description: ""
+date: 2017-09-01
+tags: [erlang, news]
+comments: false
+share: true
+---
```

Comment:
I found 1 misspelled word in this sentence: 
- ---
	- *Suggested Replacements:* -o-, -i-